packages/flutter_tools/lib/src/platform_plugins.dart

```diff
@@ -532,6 +535,78 @@ class LinuxPlugin extends PluginPlatform implements NativeOrDartPlugin {
   }
 }
 
+class CustomEmbedderPlugin extends PluginPlatform implements NativeOrDartPlugin {
+  const CustomEmbedderPlugin({
+    required this.embedderName,
+    required this.name,
+    required this.pluginPath,
+    this.dartPluginClass,
```

Comment:
Per the discussion in the doc, there's currently no approach to actually implement this, so it shouldn't be a universal key.